app/javascript/react/components/MonsterForm.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react'
+import { Redirect } from 'react-router-dom'
+
+const MonsterForm = () => {
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+  const [formData, setFormData] = useState({
+    name: "",    description: "",
+    classification: "",
+    habitat: "",
+  })
+
+  const handleChange = (event) => {
+    setFormData({
+      ...formData,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  }
+  const handleSubmit = (event) => {
+
+    event.preventDefault()
+    postNewMonster()
+  }
+
+  const postNewMonster = async () => {
+    try { 
+      const response = await fetch("/api/v1/monsters", {
+        method: "POST",
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({monster: formData})
+      })
+      if(!response.ok) {
+        throw new Error(`${response.status} (${response.statusText})`)
+      }
+      const parsedResponse = await response.json()
+      setShouldRedirect(true) 
+    } catch (error) {
+      console.error(error)
+    }
+  }
+
+  if(shouldRedirect) {
+    return <Redirect to= "/" /> 
+  }
+
+  return (
+    <div>
+      <p>{postResponse}</p>
+      <h1 className="text-center">Submit a Monster</h1>
+      <form onSubmit={handleSubmit}>
+        <label>
```

Comment:
Are there validations to ensure that a Monster name and description are provided and error messages to be displayed if invalid?